### PR TITLE
[Ready for Review] Pandas 2.0 compatibility

### LINKED ISSUE #512: Check compatability with cuDF pandas 2.0 migration
Per https://github.com/rapidsai/cudf/pull/13535 check cuxfilter for issues with Pandas 2.0.

### PR DESCRIPTION
**Pull request by AjayThorve**
fixes #512

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Ajay! 🙏

Could you please add "Fixes #512" to the OP?

File: python/cuxfilter/charts/deckgl/bindings/panel_deck.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class PanelDeck(param.Parameterized):
 
     @property
     def valid_indices(self):
-        return self.indices.intersection(self.data.index)
+        return list(self.indices.intersection(self.data.index))
```
Comment (galipremsagar) [inline python/cuxfilter/charts/deckgl/bindings/panel_deck.py:48]: I hope this conversion from device to host is an informed decision.

**Review by galipremsagar — APPROVED** [review]
Changes LGTM, one minor comment.

**Review by AyodeAwe — APPROVED** [review]

File: python/cuxfilter/charts/deckgl/bindings/panel_deck.py (hunk 1 above)
Comment (AjayThorve) [reply python/cuxfilter/charts/deckgl/bindings/panel_deck.py:48]: Ah I can see why that's confusing, but `self.data` here is actually a pandas dataframe here, which is converted from a cudf Dataframe post aggregations, for direct rendering, so technically there is no device to host conversion.

Can see the ref on L34 same file, where data is of type param.DataFrame(), which only accepts pandas as of now

Comment (galipremsagar) [reply python/cuxfilter/charts/deckgl/bindings/panel_deck.py:48]: ah, thanks for the clarification, Ajay!

**Review by exactlyallan — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Ajay and Prem! 🙏

**Pull request merged**